src/components/pages/CompanyRoles/index.tsx

```diff
@@ -43,31 +43,65 @@ export default function CompanyRoles() {
   const [topReached, setTopReached] = useState<boolean>(false)
   const { data: companyRolesResponse } = useFetchCompanyRolesQuery()
   const { data: standardLibraryResponse } = useFetchStandardLibraryQuery()
+  let filterd: StandardLibraryType = {
+    rows: [],
+    useCases: [],
+    capabilities: [],
+    roles: [],
+    status: [],
+    typesOfDocuments: [],
+  }
 
   useEffect(() => {
-    if (companyRolesResponse) {
+    if (companyRolesResponse && standardLibraryResponse) {
       if (url.indexOf('companyrolesappprovider') > 1) {
```

Comment:
I guess that should be > -1 ?
but generally use .includes(x) instead of .indexOf(x) > -1

don't use hard coded strings - use enum string types